### PR TITLE
Added LED folder and LEDSubsystem

### LINKED ISSUE #11: Add Bling status lights
Bring in the LED control code from Warcore or previous year robot.

Use LED's to display the robots current status to drivers watching.

### PR DESCRIPTION
**Pull request by warbots620**

### CODE REVIEW

File: src/main/java/frc/robot/LEDs/LEDSubsystem.java [hunk 1]
```diff
@@ -0,0 +1,200 @@
+package frc.robot.Util.LEDs;
```
Comment (m10653) [inline src/main/java/frc/robot/LEDs/LEDSubsystem.java]: Package is incorrect move file into the correct package or  change the package declaration.  (There is no Util folder)

File: src/main/java/frc/robot/LEDSubsystem.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
+package frc.robot.Util;
```
Comment (m10653) [inline src/main/java/frc/robot/LEDSubsystem.java]: There are two copies of this file?? One in src/main/java/frc/robot/LEDSubsystem.java
Other in 
src/main/java/frc/robot/LED/LEDSubsystem.java

How come?

File: src/main/java/frc/robot/LEDs/LEDIdleCommand.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    //     Color.kBlue,
+    //     Color.kBlueViolet,
+    //     Color.kPurple
+    // );
+    // protected LEDAnimation oneBallAnim = LEDAnimation.transposeBlinking(0.04, 
+    //     twoBallsAnim, 
+    //     noBallsAnim
+    // );
+
+    public LEDIdleCommand(LEDSubsystem ledSubsystem, Intake intake, FiringPins firingPins) {
```
Comment (m10653) [inline src/main/java/frc/robot/LEDs/LEDIdleCommand.java]: These Subsystems Don't exist in this years robot.

File: src/main/java/frc/robot/LEDs/LEDIdleCommand.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
+package frc.robot.Util.LEDs;
+
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Loader.Intake;
+import frc.robot.Shooter.FiringPins;
+import frc.robot.Util.LEDs.LEDSubsystem.LEDAnimation;
+import frc.robot.Util.LEDs.LEDSubsystem.LEDManager;
+
```
Comment (m10653) [inline src/main/java/frc/robot/LEDs/LEDIdleCommand.java]: Package Issues. These files do not exist in these packages so the imports fail.
There is no Util package.

**Review by m10653 — CHANGES_REQUESTED** [review]
Resolve the import/package issues, Remove the duplicate file it not needed. (If it is needed please rename it to something more understandable) 
Remove legacy imports/subsystems from last years robot.

File: src/main/java/frc/robot/LEDs/LEDSubsystem.java (hunk 1 above)
Comment (plobber) [reply src/main/java/frc/robot/LEDs/LEDSubsystem.java]: moved file into a new "util" folder and updated the package declaration

File: src/main/java/frc/robot/LEDs/LEDIdleCommand.java (hunk 3 above)
Comment (plobber) [reply src/main/java/frc/robot/LEDs/LEDIdleCommand.java]: removed irrelevant subsystems and left as a shell command

File: src/main/java/frc/robot/LEDs/LEDIdleCommand.java (hunk 4 above)
Comment (plobber) [reply src/main/java/frc/robot/LEDs/LEDIdleCommand.java]: moved file into a new "util" folder and updated the package declaration.

File: src/main/java/frc/robot/LEDSubsystem.java (hunk 2 above)
Comment (plobber) [reply src/main/java/frc/robot/LEDSubsystem.java]: @EventHorizon150 didnt know why there were two of this files

**Review by m10653 — APPROVED** [review]

**Pull request merged**